app/helpers/application_helper.rb

```diff
@@ -22,4 +22,15 @@ def actions_table_header(actions)
 
     content_tag :th, 'Actions', colspan: (actions.count if actions.count > 1)
   end
+
+  # Simple method to time a block and log its elapsed time in logs
+  def time(name, &block)
```

Comment:
I still think this method name is a bit vague. I like some combination/version of these ideas: `with_time`, `time_log`, `with_time_logging`

Also, I think this method could maybe live in a different helper file. I don't think it matters if it is the only method in the helper file. I think like `orders_helper`, `time_helper`, or `timeout_helper` might work.

Reply:
I used log_time_spent and moved it into its own helper class.